opentracing-spring-cloud/pom.xml

```diff
@@ -51,6 +51,22 @@
     </dependency>
 
     <dependency>
+      <groupId>io.opentracing.contrib</groupId>
```

Comment:
Exclusion should not change number of spans. What could change it is `TestRestTemplate`